pkg/controller/resourceclaim/controller_test.go

```diff
@@ -527,7 +527,7 @@ func TestSyncHandler(t *testing.T) {
 			}
 			defer stopInformers()
 			informerFactory.WaitForCacheSync(ctx.Done())
-			cache.WaitForCacheSync(ctx.Done(), podInformer.Informer().HasSynced, claimInformer.Informer().HasSynced, templateInformer.Informer().HasSynced)
+			cache.WaitForCacheSync(ctx.Done(), podInformer.Informer().HasSynced, podSchedulingInformer.Informer().HasSynced, claimInformer.Informer().HasSynced, templateInformer.Informer().HasSynced)
```

Comment:
A better code pattern would have been to maintain a slice of `cache.WaitForCacheSync` parameters. Then adding one more informer is a local change.

Can you change that as part of your PR?

Reply:
@pohly I removed this line because `informerFactory.WaitForCacheSync(ctx.Done())`  blocks until all started informers' caches were synced. we don't need to wait for it again.